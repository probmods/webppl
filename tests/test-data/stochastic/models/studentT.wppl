var model = function() {
  return studentT(5.0, -2.0, 3.0);
};
