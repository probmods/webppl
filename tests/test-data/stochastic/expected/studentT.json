{
  "mean": -2.0,
  "std": 3.873
}
